app/assets/stylesheets/application.css

```diff
@@ -43,15 +41,84 @@
 	font-size: 18px;
 }
 
+.fi-heart {
+  color: #DC143C; /*maroon*/
+}
+
 .nav-bar li a {
-    display: block;
-    color: black;
-    text-align: center;
-    padding: 10px;
-    /*margin-top: 20px;*/
-    text-decoration: none;
+  display: block;
+  color: black;
+  background-color: white;
+  text-align: center;
+  padding: 10px;
+  text-decoration: none;
+}
+
+.nav-bar li a:hover {
+  color: #0078a0; /* teal blue */
 }
 
 .nav-options {
 	float:right;
+}
+
+.down {
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  position: absolute;
+  text-align: right;
+  top: 400px;
+}
+
+.fi-arrow-down {
+  position: absolute;
+  right: -300px;
+  font-size: 40px;
+  color: white;
+}
+
+.top {
+	width: 100%;
```

Comment:
The indentation is inconsistent in the file, can you update so indentation is in increments of 2 spaces?